### DISCUSSION TITLE
登录问题

### DISCUSSION THREAD

**Discussion by Potat0Princess** [halo-dev/halo#5563] [General]
### System information

在主页右上角登录入口进行账号登陆demo密码异常

### What is the project operation method?

Docker

### What happened?

在主页右上角登录入口进行账号登陆demo密码异常
![屏幕截图 2024-03-21 200521](https://github.com/halo-dev/halo/assets/164336636/f48dec6f-ac11-432f-9014-d37691ec73c3)

**Comment by ruibaby** [comment]
如果默认的 demo 账号密码被修改，你可以注册一个新的账号。

此问题与 Halo 本身无关，我将转移到 Discussion。

/triage unresolved
